README.rst

```diff
@@ -104,6 +104,8 @@ Gazelle can generate Bazel BUILD files for many languages:
   generating ``java_library``, ``java_binary``, ``java_test``, and ``java_test_suite`` rules.
 
 * JavaScript / TypeScript
+  
+  Aspect provides JavaScript and TypeScript support in aspect-cli (also usable separately)
```

Comment:
This feels not quite in line with the rest of the links in this section. If you'd like to link to specific instructions for how to install Aspect's JS/TS plugins into Gazelle, I'd be happy to merge that.